### ISSUE TITLE
go install not working

### ISSUE THREAD

**Issue by cla777** [CastawayLabs/cachet-monitor#26]
I'm receiving an error on Step#2 of "How to run" trying to "go install" on Red Hat Linux.
1.I installed golang successfully and setup a golang workspace as follows:
mkdir ~/workspace
echo ‘export GOPATH=”$HOME/workspace”’ >> ~/.bashrc
source ~/.bashrc

2.Then I enter:
go install github.com/castawaylabs/cachet-monitor
and get this back:
can't load package: package github.com/castawaylabs/cachet-monitor: cannot find package github.com/castawaylabs/cachet-monitor" in any of:
/usr/lib/golang/src/github.com/castawaylabs/cachet-monitor (from $GOBOOT)"
/root/workspace/src/github.com/castawaylabs/cachet-monitor (from $GOPATH)

3.I did try this possible solution which I found while researching, but still had same problem.
mkdir /go
export GOPATH=/go
go install github.com/castawaylabs/cachet-monitor

**Comment by alexcroox** [comment]
Same issue. Never used go before but I've done the steps in 3. above too but getting

```
/go# go install github.com/castawaylabs/cachet-monitor
can't load package: package github.com/castawaylabs/cachet-monitor: cannot find package "github.com/castawaylabs/cachet-monitor" in any of:
    /usr/local/go/src/github.com/castawaylabs/cachet-monitor (from $GOROOT)
    /go/src/github.com/castawaylabs/cachet-monitor (from $GOPATH)
```

**Comment by Echobob** [comment]
Hi, thank you for raising this issue. It appears we've left a crucial step out of our guide. 

Please run `go get -d github.com/castawaylabs/cachet-monitor`
Then you should be able to run `go install github.com/castawaylabs/cachet-monitor`

Closed by #24.
